equivalences.csv

```diff
@@ -432,17 +432,45 @@ GO,GO:0048180,Activin
 GO,GO:0070652,Augmin
 GO,GO:0070765,Gamma_secretase
 GO,GO:1990779,GPIb_IX_V
+HGNC_GROUP,1000,CYP1

```

Comment:
The sorting here was by number i.e., `2` was before `1000`, this is changed by this PR, we could go back to the old ordering.